dev/fattest.simplicity/src/componenttest/topology/database/container/OracleXEContainer.java

```diff
@@ -0,0 +1,214 @@
+package componenttest.topology.database.container;
```

Comment:
Missing copyright header. Even if the original source didn't have one you should try to add something appropriate. Also, make it very clear where the original came from (github URL?) and what your changes are.